src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -264,7 +260,16 @@ func (r *ClusterReconciler) Reconcile(
 		log.Info(requeueErr.Error())
 		return ctrl.Result{RequeueAfter: requeueErr.RequeueAfter}, nil
 	}
-	return ctrl.Result{}, err
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+
+	// setting license should be at the last part as it requires AdminAPI to be running
+	if err := r.setLicense(ctx, &redpandaCluster, log); err != nil {
+		log.Info(fmt.Sprintf("setting license: %s", err.Error()))
+		return ctrl.Result{RequeueAfter: time.Minute * 1}, nil
```

Comment:
why one minute and not rely on the default backoff scheme?

Reply:
this is bound to happen at first reconciliation and wanted to avoid a lot of requeues